sql/cagg_migrate.sql

```diff
@@ -163,9 +163,20 @@ BEGIN
     _sql := format (
         $$
         WITH boundaries AS (
-            SELECT min(%1$I), max(%1$I), %1$L AS bucket_column_name, %2$L AS bucket_column_type, %3$L AS cagg_name_new
-            FROM %4$I.%5$I
-            WHERE %1$I < CAST(%6$L AS %2$s)
+            SELECT
+                min(timescaledb_information.chunks.range_start),
+                max(timescaledb_information.chunks.range_end),
+                %1$L AS bucket_column_name,
+                %2$L AS bucket_column_type,
+                %3$L AS cagg_name_new
+            FROM timescaledb_information.continuous_aggregates
+            JOIN timescaledb_information.chunks
+                ON timescaledb_information.continuous_aggregates.materialization_hypertable_schema = timescaledb_information.chunks.hypertable_schema
+                AND timescaledb_information.continuous_aggregates.materialization_hypertable_name = timescaledb_information.chunks.hypertable_name
+            WHERE
+                timescaledb_information.continuous_aggregates.view_schema = %4$L
+                AND timescaledb_information.continuous_aggregates.view_name = %5$L
+                AND timescaledb_information.chunks.range_end < CAST(%6$L AS %2$s)
```

Comment:
This will effectively omit the current chunk - but given a refresh is required, is this acceptable?